include/oboe/Definitions.h

```diff
@@ -312,6 +312,23 @@ namespace oboe {
          Allocate = AAUDIO_SESSION_ID_ALLOCATE,
     };
 
+    enum ChannelCount {
```

Comment:
Since channel count is always an int, and we want to safely cast back to an int in calculations I would suggest specifying the underlying type: 

`enum ChannelCount : int32_t {`

Reply:
this is changed now to the request format. please take a look again.  Thanks